docs/RFCS/select_from_index.md

```diff
@@ -0,0 +1,55 @@
+- Feature Name: select_from_index
+- Status: draft
+- Start Date: 2015-08-10
+- RFC PR:
+- Cockroach Issue: #2046
+
+# Summary
+
+Extend the SELECT syntax to allow an optional index to be specified in
+table references. The current table reference syntax is `table` or
+`database.table`. This syntax would be extended to allow `table.index`
+and `database.table.index`.
```

Comment:
Yeah, I think it might be wiser to use syntax like `table@index` or `table:index`.